internal/plugin/config_plugin.go

```diff
@@ -23,14 +25,12 @@ import (
 	"github.com/nginx/agent/v3/internal/service"
 )
 
-// remove when tenantID is being set
-const tenantID = "7332d596-d2e6-4d1e-9e75-70f91ef9bd0e"
-
 type Config struct {
 	messagePipe    bus.MessagePipeInterface
 	configServices map[string]service.ConfigServiceInterface
 	resource       *v1.Resource
 	agentConfig    *config.Config
+	configClient   client.ConfigClient
```

Comment:
is a better name for this fileServiceClient?

Reply:
I can change it if needed, I was just reusing the naming we had previously for the http config client dont have a strong opinion

Reply:
Just to clarify do ye also want to rename the ConfigClient interface ? Because we already have a FileServiceClient in the protos

Reply:
Is it just the same gRPC service that implements that interface? So essentially `FileServiceClient` and `ConfigClient` have the same implementation?

```go
// FileServiceClient is the client API for FileService service.
//
// For semantics around ctx use and closing/ending streaming RPCs, please refer to https://pkg.go.dev/google.golang.org/grpc/?tab=doc#ClientConn.NewStream.
type FileServiceClient interface {
	// Get the overview of files for a particular configuration version of an instance
	GetOverview(ctx context.Context, in *GetOverviewRequest, opts ...grpc.CallOption) (*GetOverviewResponse, error)
	// Update the overview of files for a particular set of file changes on the data plane
	UpdateOverview(ctx context.Context, in *UpdateOverviewRequest, opts ...grpc.CallOption) (*UpdateOverviewResponse, error)
	// Get the file contents for a particular file
	GetFile(ctx context.Context, in *GetFileRequest, opts ...grpc.CallOption) (*GetFileResponse, error)
	// Update a file from the Agent to the Server
	UpdateFile(ctx context.Context, in *UpdateFileRequest, opts ...grpc.CallOption) (*UpdateFileResponse, error)
}

type ConfigClient interface {
	GetOverview(ctx context.Context, request *v1.GetOverviewRequest) (*v1.FileOverview, error)
	GetFile(ctx context.Context, request *v1.GetFileRequest) (*v1.FileContents, error)
}
```

Reply:
Okay I propose we don't rename it now, seeing as we are looking into consolidating some of our services (and their naming too), so the client is likely subject to change in the near future. No need to block this PR over something so small.

Reply:
Sounds good, I'll make note of it somewhere.